packages/accounts/src/light-account/utils.ts

```diff
@@ -43,6 +44,10 @@ export const supportedChains: Chain[] = [
   base,
   baseGoerli,
   baseSepolia,
+  fraxtal,
+  fraxtalTestnet,
+  zora,
+  zoraSepolia,
```

Comment:
we should add these to aa-core as alchemy chains, and use those. even if they are not alchemy rpc url, can we configure these in aa-core with Blast url? if not, let's at least add comment on why these are from viem and others are not.

Reply:
Would adding a non-Alchemy URL would break things because we assume we can append an Alchemy API key to the URLs

Reply:
Added to aa-core without an alchemy base url